packages/cozy-stack-client/src/xhrFetch.js

```diff
@@ -16,7 +16,10 @@ const headersFromString = headerString => {
  * @private
  * @param {string} fullpath - Route path
  * @param {object} options - Fetch options
- * @param {Function} options.onUploadProgress - Callback to receive upload progress events
+ * @param {(this: XMLHttpRequestUpload, ev: ProgressEvent<XMLHttpRequestEventTarget>) => any} options.onUploadProgress - Callback to receive upload progress events
+ * @param {string} options.method - TODO
```

Comment:
I don't know if TS comes with built-in type for this, but can be one of those https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods